code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -43,23 +43,23 @@
 	var/mob/living/carbon/human/friend
 	var/tamed = 0
 
-/mob/living/simple_animal/otie/fluff
+/mob/living/simple_animal/otie/fluff //gets the pet2tame feature. starts out hostile tho so get gamblin'
```

Comment:
Yeah should've probably changed it. Just sort of based it on the Runtime(the cat) befriending code that used the term.